torch/_torch_docs.py

```diff
@@ -6163,6 +6163,35 @@ def merge_dicts(*dicts):
             [1, 2, 2]])
 """.format(**factory_common_args))
 
+add_docstr(torch.true_divide,
+           r"""
+true_divide(dividend, divisor) -> Tensor
+
+Performs "true division" that always computes the dividend divided by the divisor
```

Comment:
nit: "dividend divided by the divisor" is pretty wordy, I'd probably just say "computes the division in floating point"